meta-lmp-bsp/dynamic-layers/xilinx-tools/recipes-bsp/bitstream/bitstream-extraction_git.bb

```diff
@@ -0,0 +1,30 @@
+DESCRIPTION = "Recipe to extract bitstream"
+
+LICENSE = "MIT"
+LIC_FILES_CHKSUM = "file://${COMMON_LICENSE_DIR}/MIT;md5=0835ade698e0bcf8506ecda2f7b4f302"
+
+DEPENDS += "virtual/hdf"
+
+PROVIDES = "virtual/bitstream"
+
+PACKAGE_ARCH ?= "${MACHINE_ARCH}"
+
+inherit xsctbit
+
+XSCTH_MISC = "-hwpname ${XSCTH_PROJ}_hwproj -hdf_type ${HDF_EXT}"
+
+do_compile[noexec] = "1"
+
+SYSROOT_DIRS += "${nonarch_base_libdir}/firmware"
+
+do_install() {
+
+    if [ -e ${XSCTH_WS}/${XSCTH_PROJ}_hwproj/*.bit ]; then
+        install -d ${D}/${nonarch_base_libdir}/firmware/
+        install -d ${D}/var/lib/firmware/bitstream/
+        ln -s -r ${D}/var/lib/firmware/bitstream ${D}/${nonarch_base_libdir}/firmware/bitstream
+        install -Dm 0644 ${XSCTH_WS}/${XSCTH_PROJ}_hwproj/*.bit ${D}/${nonarch_base_libdir}/firmware/
+    fi
+}
+
```

Comment:
For loading from fit you will also need a do_deploy (no needed for now, but just a heads up).

Reply:
understood, I will add that when I look at the u-boot pieces